app/controllers/events/draft.js

```diff
@@ -0,0 +1,100 @@
+import Ember from 'ember';
+
+const { Controller, String } = Ember;
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'name',
+      template     : 'components/ui-table/cell/cell-event',
+      title        : 'Name'
+    },
+    {
+      propertyName : 'startTime',
+      template     : 'components/ui-table/cell/cell-date',
+      title        : 'Date'
+    },
+    {
+      propertyName : 'roles',
+      template     : 'components/ui-table/cell/cell-roles',
+      title        : 'Roles'
+    },
+    {
+      propertyName : 'sessions',
+      template     : 'components/ui-table/cell/cell-sessions',
+      title        : 'Sessions'
+    },
+    {
+      propertyName : 'speakers',
+      title        : 'Speakers'
+    },
+    {
+      propertyName : 'tickets',
+      template     : 'components/ui-table/cell/cell-tickets',
+      title        : 'Tickets'
+    },
+    {
+      propertyName : 'url',
+      title        : 'Public URL',
+      class        : 'hide'
+    },
+    {
+      template : 'components/ui-table/cell/cell-buttons',
+      title    : 'Actions'
+    }
+
+  ],
+  data: [{
+    name      : 'Sample_event',
+    date      : String.htmlSafe('<span>June 6, 2016 - 05:30 PM <br> To  <br>\n June 20, 2016 - 05:30 PM</span>'),
+    startAt   : 'June 6, 2016 - 05:30 PM',
+    endAt     : 'June 20, 2016 - 05:30 PM',
+    roles     : 'john_doe@gmail.com',
+    tickets   : String.htmlSafe(''),
+    sessions  : String.htmlSafe(''),
+    speakers  : '3',
+    url       : String.htmlSafe('<a href="">Sample event url</a>'),
```

Comment:
Would just be a normal string. Never use `String.htmlSafe` unless there is no other option. Avoid it like the plague. This should be handled in the template